lib/main.js

```diff
@@ -473,6 +476,7 @@ Lambda.prototype._uploadExisting = (lambda, params) => {
         'Runtime': params.Runtime,
         'VpcConfig': params.VpcConfig,
         'Environment': params.Environment,
+        'KMSKeyArn': params.KMSKeyArn,
```

Comment:
I wasn't sure if I should include a conditional check here or not to add KMSKeyArn to the params.